README.md

```diff
@@ -261,6 +262,18 @@ Errors are logged in the same format to `stderr`, with the `Error` property set:
 }
 ```
 
+### Filtering
+
+By setting the environment variable `EMAIL_DENY_REGEX`, the relay will not
+allow the emails to be sent if the `from` field matches a specific regex.
+
+The following example will prevent emails being sent when the `from` email
+ends with `com`.
+
+```shell script
+export EMAIL_DENY_REGEX="(.*.com)"
```

Comment:
My comment here would be to demo a more complex real-world example - i.e. `(.*@example.com$)|(test@test.com)`